pkg/election/lease.go

```diff
@@ -154,11 +154,14 @@ func (l *lease) keepAliveWorker(ctx context.Context, interval time.Duration) <-c
 
 		log.Info("start lease keep alive worker", zap.Duration("interval", interval), zap.String("purpose", l.Purpose))
 		defer log.Info("stop lease keep alive worker", zap.String("purpose", l.Purpose))
-
+		lastTime := time.Now()
 		for {
-			go func() {
+			start := time.Now()
+			if start.Sub(lastTime) > interval*2 {
+				log.Warn("the interval between keeping alive lease is too long", zap.Time("last-time", lastTime))
```

Comment:
Will it print a lot?

Reply:
It's possible. But similar to the 'clock offset' warning log in tso, it is indeed a worthwhile log to output